api/mesh/v1alpha1/dataplane_helpers.go

```diff
@@ -288,6 +288,15 @@ func (n *Dataplane_Networking) GetInboundForPort(port uint32) *Dataplane_Network
 	return nil
 }
 
+func (n *Dataplane_Networking) GetInboundForSectionName(sectionName string) *Dataplane_Networking_Inbound {
+	for _, inbound := range n.Inbound {
+		if inbound.Name == sectionName || strconv.Itoa(int(inbound.Port)) == sectionName {
```

Comment:
again this complex code. Any chance we can use something generic in `inbound` ?

Reply:
made some refactoring, this should be better now